bin/kano-shutdown

```diff
@@ -10,25 +10,26 @@
 # /sbin/poweroff, reboot, and loginctl.
 #
 
-# The steps needed before showing the dialog are a bit time consuming
-# so we raise the hourglass as soon as possible
-from kdesk.hourglass import hourglass_start, hourglass_end
-hourglass_start('kano-shutdown')
 
 from gi.repository import Gtk, Gdk
 
 from kano.gtk3.buttons import KanoButton
 from kano.gtk3.application_window import ApplicationWindow
 from kano.gtk3.apply_styles import apply_styling_to_screen
-from kano.utils import run_cmd, run_bg
 
 from kano.paths import common_css_dir
+from kano.utils import run_cmd, run_bg
+
+# The steps needed before showing the dialog are a bit time consuming
+# so we raise the hourglass as soon as possible
+from kdesk.hourglass import hourglass_start, hourglass_end
+hourglass_start('kano-shutdown')
```

Comment:
I tested that and it doesn't make a difference, there are no lengthy imports here.